queries.sql

```diff
@@ -29,3 +29,72 @@ where name != 'Gabumon';
 select *
 from animals
 where weight_kg between 10.4 and 17.3;
+
+begin;
```

Comment:
![image](https://github.com/akezeth/Vet-clinic-database/assets/93039359/00b70dbd-7c34-4bca-bce5-072d20d22648)

**[OPTIONAL]**

- [ ] It would be nice if you fix your linkedin link because it is broken 😨 Kindly fix it to make your readme descriptive and professional 👍

Reply:
Thanks for your comment, I have fixed it.